src/autocomplete.js

```diff
@@ -338,11 +360,10 @@ class Autocomplete extends Component {
   //\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ ITEM
 
   reset = () => {
-    this.setState(({selectedItem}) => ({
+    this.setState({
       isOpen: false,
       highlightedIndex: null,
-      value: this.getValue(selectedItem),
```

Comment:
The `inputValue` should be reset to the `getValue` of the currently selected item right?